applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -181,10 +178,10 @@ void KratosStructuralMechanicsApplication::Register()
     KRATOS_REGISTER_VARIABLE( MEAN_RADIUS )
     KRATOS_REGISTER_VARIABLE( SECTION_SIDES )
     KRATOS_REGISTER_VARIABLE( GEOMETRIC_STIFFNESS )
-	KRATOS_REGISTER_VARIABLE(LOCAL_AXIS_VECTOR_1)
-	KRATOS_REGISTER_VARIABLE(LOCAL_ELEMENT_ORIENTATION)
-	KRATOS_REGISTER_VARIABLE(ORTHOTROPIC_ORIENTATION_ASSIGNMENT)
-	KRATOS_REGISTER_VARIABLE(ORTHOTROPIC_FIBER_ORIENTATION_1)
+    KRATOS_REGISTER_VARIABLE(LOCAL_AXIS_VECTOR_1)
```

Comment:
do not define LOCAL_AXIS_VECTOR_1

use LOCAL_AXIS_1 instead.

but mostly...why are u adding these variables or touching this file in this PR??